src/App.js

```diff
@@ -7,27 +7,28 @@ import {
   Route,
 } from 'react-router-dom';
 import apolloClient from './apollo';
-import English from './English';
-import Portuguese from './Portuguese';
-import Spanish from './Spanish';
-import Home from './Home';
+import Home from './Home/Home';
+import Login from './Login/Login';
+import Register from './Register/Register';
+import NotFound from './NotFound/NotFound';
+import PrivateRoute from './PrivateRoute';
 
 function App() {
   return (
     <ApolloProvider client={apolloClient}>
       <Router>
         <Switch>
-          <Route path="/home">
+          <PrivateRoute exact path="/">
```

Comment:
I really like this idea of having a `PrivateRoute` for pages only accessible to logged in users 👍